code/_onclick/click.dm

```diff
@@ -96,7 +96,8 @@
 		return
 
 	if(modifiers["right"]) //CIT CHANGE - allows right clicking to perform actions
-		RightClickOn(A,params) //CIT CHANGE - ditto
+		if(combatmode)
+			RightClickOn(A,params) //CIT CHANGE - ditto

```

Comment:
This isn't really necessary. It should be up to to individual right click overrides to determine their behavior when combat mode is disabled, instead of right clicking simply being disabled entirely when combat mode is disabled

Reply:
RightClickOn() is mostly a copypaste of ClickOn() minus the modifiers and some added special interactions seemengly purely intended for combat mode.
At this point, I'll just move the combatmode check to the line above so it won't return and instead process like an average click.